### PR TITLE
Migrate to 1ES templates

### PR DESCRIPTION
**Pull request by joeloff**

### CODE REVIEW

**Review by nagilson — COMMENTED** [review]
I think we should delete the code ql pipeline if we arent going to use it anymore

**Comment by joeloff** [comment]
> I think we should delete the code ql pipeline if we arent going to use it anymore

Deleted

File: azure-pipelines.yml [hunk 1]
```diff
@@ -38,177 +37,187 @@ variables:
...
+extends:
+  ${{ if notin(variables['Build.Reason'], 'PullRequest') }}:
+    template: v1/1ES.Official.PipelineTemplate.yml@1esPipelines
+  ${{ else }}:
+    template: v1/1ES.Unofficial.PipelineTemplate.yml@1esPipelines
+  parameters:
+    sdl:
+      sourceAnalysisPool:
+        name: $(DncEngInternalBuildPool)
+        image: 1es-windows-2022-pt
```
Comment (marcpopMSFT) [inline azure-pipelines.yml:67]: fyi, you don't have to use the -pt images anymore and in fact, they will go away eventually.

**Review by marcpopMSFT — APPROVED** [review]

File: azure-pipelines.yml (hunk 1 above)
Comment (joeloff) [reply azure-pipelines.yml:67]: I'll fix that up

**Pull request merged**